flink-table-store-benchmark/flink-table-store-micro-benchmarks/src/main/java/org/apache/flink/table/store/benchmark/file/mergetree/MergeTreeBenchmark.java

```diff
@@ -150,13 +149,19 @@ private RecordWriter<KeyValue> recreateMergeTree(
                             @Override
                             public List<DataField> keyFields(TableSchema schema) {
                                 return Collections.singletonList(
-                                        new DataField(0, "k", new AtomicDataType(new IntType())));
+                                        new DataField(
```

Comment:
Will Table Store get rid of `RowType` and `org.apache.flink.table.types.logical.IntType` later?

Reply:
Yes, we will